### PR TITLE
feat: composable for lastTransactions

### PR DESCRIPTION
**Pull request by konradodo**

### CODE REVIEW

File: src/composables/lastTransactions.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { computed } from '@vue/composition-api';
+import { ITransaction } from '../types';
+import store from '../store';
+import {
+  COUNT_OF_DASHBOARD_TRANSACTION,
+  sortTransaction,
+} from '../popup/utils';
+
+export const useLastTransaction = () => {
```
Comment (pstachel) [inline src/composables/lastTransactions.ts:9]:
```suggestion
export const useCombinedLatestTransactions = () => {
```

File: src/composables/lastTransactions.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  const availableTokens = computed<Record<string, ITransaction[]>>(
+    () => (store as any).state?.fungibleTokens?.transactions,
+  );
```
Comment (pstachel) [inline src/composables/lastTransactions.ts:12]: you have set either wrong name of variable or have wrong getter

Comment (pstachel) [reply src/composables/lastTransactions.ts:12]: Also please use useState here

File: src/composables/lastTransactions.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  const lastTransactions = computed(() => {
```
Comment (pstachel) [inline src/composables/lastTransactions.ts:13]:
```suggestion
  const latestTransactions = computed(() => {
```

File: src/composables/lastTransactions.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    const allTransaction = Object.values(availableTokens.value).flatMap((t) => t);
```
Comment (pstachel) [inline src/composables/lastTransactions.ts:14]:
```suggestion
    const allTransactions = Object.values(availableTokens.value).flatMap((t) => t);
```

File: src/popup/App.vue [hunk 5]
```diff
@@ -20,7 +20,7 @@
       :class="{ 'show-scrollbar': $route.meta.showScrollbar }"
     >
       <Header v-if="showHeader" />
-
+      {{ lastTransactions }}
```
Comment (pstachel) [inline src/popup/App.vue:23]: shouldn't be here

**Review by pstachel — CHANGES_REQUESTED** [review]

**Review by pstachel — COMMENTED** [review]
Also I feel that there's no need for a composable here - combined transaction list will be used only as a separate component on dashboard, so whole logic can be encapsulated there

File: src/composables/lastTransactions.ts (hunk 2 above)
Comment (konradodo) [reply src/composables/lastTransactions.ts:12]: i can't, after @peronczyk  change we cant use useState, useGetter in composables

**Pull request closed without merging**